src/background.ts

```diff
@@ -447,8 +457,16 @@ async function createWindow() {
   }
   if (isDevelopment) win.webContents.openDevTools();
 
+  if (isMac) nativeTheme.themeSource = "light";
```

Comment:
ワークアラウンドなので意図をコメントすると良いかもと思いました。こんな感じでしょうか
```suggestion
  // darkモードだとMacの閉じるボタンなどが見えなくなるのでlightモードに固定
  if (isMac) nativeTheme.themeSource = "light";
```

Reply:
わかりました！